annotation-file-utilities/build.gradle

```diff
@@ -716,20 +710,12 @@ task pythonIsInstalled(type: Exec) {
 }
 
 
-task setLocalRepo(type:Exec) {
-    commandLine 'git', 'worktree', 'list'
-    standardOutput = new ByteArrayOutputStream()
-    doLast {
-       String worktreeList = standardOutput.toString()
-       localRepo = worktreeList.substring(0, worktreeList.indexOf(" ")) + "/.git"
-    }
-}
-
-task installGitHooks(type: Copy, dependsOn: 'setLocalRepo') {
+task installGitHooks(type: Copy) {
     description 'Copies git hooks to .git directory'
-    from files(".git.pre-commit")
+    from files("../.git.pre-commit")
     rename('.git\\.(.*)', '$1')
-    into localRepo + "/hooks"
```

Comment:
The localRepo property is set to `.git` and configuration time, so this always evaluates to `.git/hooks` which is incorrect for AFU.

Reply:
I figured out how to do this correctly.